pyproject.toml

```diff
@@ -47,19 +47,19 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = "^3.7.0"
+python = "^3.6.0"
 aiohttp = "^3.5"
 asgiref = "^3.2"
 channels = "^2.2"
-django = "^2.2"
+django = ">=2.2"
 graphene = "^2.1"
 graphql-core = "^2.2"
 msgpack = "^0.6.1"
 
 
 [tool.poetry.dev-dependencies]
 bandit = "*"
-black = { version = "*", allows-prereleases = true }
```

Comment:
Again, why? What is the purpose of these changes?

Reply:
This was found inside travis.

Poetry 1.0 changes the field name: https://github.com/python-poetry/poetry/releases/tag/1.0.0

> Specifying dependencies with `allows-prereleases` in the pyproject.toml file is deprecated for consistency with the add command. Use `allow-prereleases` instead.

Newer poetry versions renamed `allow-prereleases` to `allow-prereleases`. This caused the travis build to fail.

It also updates black.

https://github.com/datadvance/DjangoChannelsGraphqlWs/pull/37/commits/7a772583d3421f89f978ea0639ae688c50e549d3

![image](https://user-images.githubusercontent.com/26336/78461335-99bd2e80-768d-11ea-9d0f-898e8d932f59.png)

Reply:
https://travis-ci.org/github/datadvance/DjangoChannelsGraphqlWs/jobs/626173702?utm_medium=notification&utm_source=github_status

![image](https://user-images.githubusercontent.com/26336/78461507-38965a80-768f-11ea-9a75-c162916b932d.png)

```
The "black" dependency specifies the "allows-prereleases" property, which is deprecated. Use "allow-prereleases" instead.
ImportError while loading conftest '/home/travis/build/datadvance/DjangoChannelsGraphqlWs/tests/conftest.py'.
tests/conftest.py:27: in <module>
    import channels
E   ModuleNotFoundError: No module named 'channels'
```

(Thankfully travis still has these!)